src/core.c/Main.pm6

```diff
@@ -302,7 +302,33 @@ my sub RUN-MAIN(&main, $mainline, :$in-as-argsfiles) {
                     }
                     @positional.push($argument);
                 }
-                @arg-help.push($argument => $param.WHY.contents) if $param.WHY and (@arg-help.grep:{ .key eq $argument}) == Empty;  # Use first defined
+            		#@arg-help.push($argument => $param.WHY.contents) if $param.WHY and (@arg-help.grep:{ .key eq $argument}) == Empty;  # Use first defined
+                if $param.WHY and (@arg-help.grep:{ .key eq $argument}) == Empty {
+                    my $why = $param.WHY.contents; # Use first defined
+                    if $param.default -> $d {
+                          constant MAXCHARS = 20;
+                          # $middle is for long integers and other argument types
+                          # that are better split in the middle.
+                          if ( my $def = $d() ).defined {
```

Comment:
Perhaps better written as `with $d() -> $def {` ?

Reply:
Done specifically to handle when a value is defined, but Falsey (ala 0, False)

Reply:
@Xliff that's exactly what you get with `with`: `my $a = 0; say $_ with $a`